scripts/acnv_cromwell_tests/case_input_wgs.tsv

```diff
@@ -1,2 +1,2 @@
-T1	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-t1-chr20-downsampled.deduplicated.bam	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-t1-chr20-downsampled.deduplicated.bam.bai	N1	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-n1-chr20-downsampled.deduplicated.bam	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-n1-chr20-downsampled.deduplicated.bam.bai
-T2	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-t2-chr20-downsampled.deduplicated.bam	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-t2-chr20-downsampled.deduplicated.bam.bai	N2	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-n2-chr20-downsampled.deduplicated.bam	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-n2-chr20-downsampled.deduplicated.bam.bai
+T1	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143-t1-chr20-downsampled.deduplicated.bam	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143-t1-chr20-downsampled.deduplicated.bam.bai	N1	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-n1-chr20-downsampled.deduplicated.bam	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-n1-chr20-downsampled.deduplicated.bam.bai
+T2	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143-t2-chr20-downsampled.deduplicated.bam	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143-t2-chr20-downsampled.deduplicated.bam.bai	N2	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-n2-chr20-downsampled.deduplicated.bam	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-n2-chr20-downsampled.deduplicated.bam.bai
```

Comment:
Ah, I see---so they're from the normal?  Hmm, I still think having them named after the normal is confusing, but naming them after the tumor would be a white lie...

Let's go ahead and make the name change, then plan on actually replacing them with a snippet of tumor (ideally containing a few events) later.  I'll file an issue.